client/js/app/stores/ExplorerStore.js

```diff
@@ -283,6 +283,10 @@ function _wrapGroupBy(group_by) {
   return group_by;
 }
 
+function _coerceToNumber(str) {
+  return str && typeof(str) === 'string' ? Number(str) : str;
```

Comment:
Thanks for reviewing @aroc. I included the conditional to account for a situation where an empty string is passed to `_coerceToNumber`. We're currently using an empty string as the default percentile value, which will cause the query to error if executed. Without the conditional, `Number("")` returns `0` which will successfully return unexpected query results to the user (when there should probably be an error).

It looks like there is another issue with validation right now, making it possible for a user to execute a percentile query without entering a percentile value in the input field, actualizing this edge case. I'm happy to look into that too (perhaps as a separate PR). Thoughts?